SETUP.md

```diff
@@ -107,7 +107,7 @@ properties_encryption_key: ''
 
 ### macOS
 
-These steps are for Apple devices running **macOS Monterey and Ventura**, including those running on [Apple Silicon (M1|M2)](https://en.wikipedia.org/wiki/Apple_silicon#M_series). 
+These steps are for Apple devices running **macOS Monterey and Ventura**, including those running on [Apple Silicon (M1|M2|M3)](https://en.wikipedia.org/wiki/Apple_silicon#M_series). 
```

Comment:
Might be worth it for us to specify a specific major release number of macOS instead of their friendly product names.

```suggestion
These steps are for Apple devices running **macOS 14.x**, including those running on [Apple Silicon (M1|M2|M3)](https://en.wikipedia.org/wiki/Apple_silicon#M_series). 
```